Core/SchemeHandler.swift

```diff
@@ -68,7 +69,7 @@ public class SchemeHandler {
         }
 
         switch PlatformScheme(rawValue: schemeString) {
-        case .sms, .mailto, .itms, .itmss, .itunes, .itmsApps, .itmsAppss:
+        case .sms, .mailto, .itms, .itmss, .itunes, .itmsApps, .itmsAppss, .shortcuts:
```

Comment:
There is also `shortcuts-production` and `workflow`

Reply:
Great find, thanks.